### DISCUSSION TITLE
Analog television (Low power or far)

### DISCUSSION THREAD

**Discussion by Degamisu** [ntsc-rs/ntsc-rs#151] [Presets]
I tried to get the most accurate result of a low power transmitter, let me know if you want the preset.

![image](https://github.com/user-attachments/assets/6cdcef03-39fb-4306-9cbe-280d008f6652)
![image](https://github.com/user-attachments/assets/1c4953f8-ebc3-4999-8381-42776551bb7d)

**Comment by fuki98** [comment]
Can I have it please?

**Reply by Degamisu** [reply]
Sure!

[Low-Power-NTSC.json](https://github.com/user-attachments/files/16561262/Low-Power-NTSC.json)

**Reply by fuki98** [reply]
Thank you!

**Comment by smfthe1199** [comment]
This will be added to the next version of the **NTSC-RT Preset Pack** alongside of several others.

I got permission from the creator of this pack that I can upgrade the pack.

**Reply by Degamisu** [reply]
Can't wait for these to be released! I'll try and pump out a few more presets for you guys.

**Reply by fuki98** [reply]
Great
